pkgs/applications/misc/zathura/wrapper.nix

```diff
@@ -11,7 +11,7 @@ symlinkJoin {
   in ''
     makeWrapper ${zathura_core.bin}/bin/zathura $out/bin/zathura \
       --prefix PATH ":" "${lib.makeBinPath [ file ]}" \
-      --add-flags --plugins-dir="$out/lib/zathura"
+      --set ZATHURA_PLUGINS_PATH "$out/lib/zathura"
```

Comment:
I wonder if I wouldn't use --prefix instead to let the possibility to override it.

Reply:
and if we keep `--plugins-dir`, can ZATHURA_PLUGINS_PATH override it, i.e., why did you change this ?

Reply:
:+1: for `--prefix`, regarding the change, to quote my commit message:

```


As soon as you open another window in zathura from zathura (e.g.
to follow a PDF-embedded link to another PDF), it currently fails
because --plugins-dir= isn't passed along. This is because zathura
uses argv[0][1] to open another process and the GApps-wrapper inside
zathuraPkgs.zathura_core does a exec -a "$0" which isn't the
bin/zathura from the wrapper that appends --plugins-dir=:

execve("/nix/var/nix/profiles/per-user/ma27/home-manager-140-link/home-path/bin/zathura", ["/nix/var/nix/profiles/per-user/ma27/home-manager-140-link/home-path/bin/zathura", "/home/ma27/Documents/Uni/Studium/Notepad/Aktuell/Index.pdf"], 0x7ffd7a4bf4b0 /* 108 vars */) = 0
execve("/nix/store/2wjhxbgzcnn0qqdwqy0m01hw39dxwfmk-zathura-0.4.8-bin/bin/zathura", ["/nix/store/2wjhxbgzcnn0qqdwqy0m01hw39dxwfmk-zathura-0.4.8-bin/bin/zathura", "--plugins-dir=/nix/store/wcch63yzwykc9x5393dzjfdxsf80mrb8-zathura-with-plugins-0.4.8/lib/zathura", "/home/ma27/Documents/Uni/Studium/Notepad/Aktuell/Index.pdf"], 0x1442010 /* 107 vars */) = 0
execve("/nix/store/2wjhxbgzcnn0qqdwqy0m01hw39dxwfmk-zathura-0.4.8-bin/bin/.zathura-wrapped", ["/nix/store/2wjhxbgzcnn0qqdwqy0m01hw39dxwfmk-zathura-0.4.8-bin/bin/zathura", "--plugins-dir=/nix/store/wcch63yzwykc9x5393dzjfdxsf80mrb8-zathura-with-plugins-0.4.8/lib/zathura", "/home/ma27/Documents/Uni/Studium/Notepad/Aktuell/Index.pdf"], 0x81d010 /* 108 vars */) = 0

I figured it's way simpler to actually use the environment variable
ZATHURA_PLUGINS_PATH for that purpose as the environment is
inherited when a new process is started.

```